data/stats/get_runway_stats.py

```diff
@@ -0,0 +1,39 @@
+"""
+    Gather runway statistics for each 
+    airport from given airport runway data. 
+"""
+
+from dataclasses import dataclass, field
+import pandas as pd
+from src.data.types import Runway
+
+@dataclass
+class RunwayStats:
+    runways: pd.DataFrame
+    count: pd.DataFrame = field(init=False)
+    len: pd.DataFrame = field(init = False)
+
+    def __post_init__(self) -> None:
+        self.count = pd.DataFrame(self.runways.map(self.count_runways))
+        self.len = pd.DataFrame(self.runways.map(self.sum_runways_len))
+
+    def count_runways(self, airport_runways: list[Runway|None]) -> int | None:
+        runway_counter: int|None
+        runway_counter = len(airport_runways)
+        if runway_counter == 0:
+            runway_counter = None
+        return runway_counter
```

Comment:
I think this could be written more simply as 

```python
def count_runways(self, airport_runways: list[Runway]) -> int | None:
    return len(airport_runways) or None
```